tap_clickup/streams.py

```diff
@@ -16,7 +60,9 @@ class TeamsStream(ClickUpStream):
     primary_keys = ["id"]
     replication_key = None
     schema_filepath = SCHEMAS_DIR / "team.json"
-    records_jsonpath = "$.teams[*]"
+    # Necessary because if you have access to multiple workteams, the responses are replicated N times where N = # of worspaces you have access to
+    records_jsonpath = f"$.teams[?(@.id == {workteamid})]" 
```

Comment:
Did they add a new concept of workteamid? Or workspace_id, could you link to that in clickups docs?

Reply:
So the structure is Workspace>Space>Folder>List>Task>Subtask

In the app/platform they are called workspaces, but for the API they are called Teams, also there is a feature called "teams" (that group users together) but in the API they call it "group"

![image](https://github.com/AutoIDM/tap-clickup/assets/63605974/8a5b9e34-b661-4a1c-8130-bb4c5f1ac0cd)

https://clickup.com/api/clickupreference/operation/CreateTeam/